chapters/CRUD-Operations.md

```diff
@@ -81,26 +88,30 @@ It returns the following outputs:
 The Read operation is an algorithm consisting of a series of subroutine algorithms
 executed by a resolver after a resolution request identifying a specific
 **did:btc1** `identifier` is received from a client at ::Resolution Time::.
-The request MUST always contain the resolutionOptions object containing additional
+The request MUST always contain the `resolutionOptions` object containing additional
 information to be used in resolution. This object MAY be empty. 
```

Comment:
```suggestion
The request MUST always contain the `resolutionOptions` object containing additional
information to be used in resolution. This object MAY be empty. 
See the [DID Resolution specification](https://w3c.github.io/did-resolution/#did-resolution-options) for further details about the DID Resolution Options object.
```

I am sure there are many places throughout the spec that could benefit from better referencing. This seems like an easy one to catch now